### ISSUE TITLE
Alias expansion in match

### ISSUE THREAD

**Issue by jjl** [OvermindDL1/protocol_ex#11]
It appears that aliases are not respected during expansion.

```elixir
alias Bricks.Done
alias Bricks.Client.Simple
alias Bricks.Socket.Tcp

defimplEx DoneSimpleTcp, {%Bricks.Client.Simple{}, %Bricks.Socket.Tcp{}}, for: Done do
  def return({client, tcp}), do: :gen_tcp.close(tcp.socket)
  def errored({client, tcp}), do: :ok
  def close({client, tcp}), do: :gen_tcp.close(tcp.socket)
end
```

Here, I should be able to refer to `Bricks.Client.Simple` as `Simple` and `Bricks.Socket.Tcp` as `Tcp` in the match, but it only works if I fully qualify them. Note that this only seems to affect the match, the `Done` works fine

**Comment by OvermindDL1** [comment]
Ah!  Indeed they are not because I copy the AST unchanged.  I'll need to iterate over the entire AST tree but it is possible...

**Comment by OvermindDL1** [comment]
Can you ping me tomorrow?  I may be able to do it then.

**Comment by OvermindDL1** [comment]
Thanks much!

Should be fixed by 0bc3636 and version 0.3.16 pushed.  :-)
